vagrant-spk

```diff
@@ -105,6 +108,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provider :virtualbox do |vb, override|
     vb.cpus = cpus
     vb.memory = assign_ram_mb
+    vb.name = PROJECT_BASENAME + "_sandstorm"
```

Comment:
Let me test this now, to see it causes duplicates.

IMHO we should do the same as for `libvirt`.

I propose you calculate that string in a variable, rather than do it once per `config.vm.provider`.